src/theory/quantifiers/global_negate.cpp

```diff
@@ -0,0 +1,112 @@
+/*********************                                                        */
+/*! \file global_negate.cpp
+ ** \verbatim
+ ** Top contributors (to current version):
+ **   Andrew Reynolds
+ ** This file is part of the CVC4 project.
+ ** Copyright (c) 2009-2017 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved.  See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** \brief Implementation of global_negate
+ **/
+
+#include "theory/quantifiers/global_negate.h"
+#include "theory/rewriter.h"
+
+using namespace std;
+using namespace CVC4::kind;
```

Comment:
Remove, since you always use `kind::<KIND>` (except for  BOUND_VARIABLE).

Reply:
I kept and replaced the kind:: in the file.  (I generally prefer using this namespace, I had mistakenly used kind::).